package.json

```diff
@@ -36,7 +36,7 @@
         "css-loader": "^0.25.0",
         "eslint": "^3.0.0",
         "eslint-loader": "^1.0.0",
-        "eslint-plugin-react": "^6.0.0",
+        "eslint-plugin-react": "^6.7.0",
```

Comment:
I was trying to fix a `jsx-uses-vars` linting error and I forced it up to 6.7 to see if that helped. Plus I was mixed up between `~` and `^` so I'll revert this